packages/primitives/test/emotion-primitives.test.js

```diff
@@ -188,18 +188,86 @@ describe('Emotion primitives', () => {
   })
 
   test('custom shouldForwardProp works', () => {
-    const Text = styled.Text``
-    const Title = props => <Text {...props} />
-    // $FlowFixMe
+    let forwardedProps: any
+    const Title = props => {
+      forwardedProps = props
+      return null
+    }
     const StyledTitle = styled(Title, {
       shouldForwardProp: prop => prop !== 'color' && prop !== 'theme'
     })`
       color: ${props => props.color};
     `
 
-    const tree = renderer
-      .create(<StyledTitle color="hotpink">{'Emotion'}</StyledTitle>)
-      .toJSON()
-    expect(tree).toMatchSnapshot()
+    renderer.create(
+      // $FlowFixMe
+      <StyledTitle color="hotpink" disabled>
```

Comment:
I changed these tests because the Text component has it's own filters, so the test would always pass even if you added more props